.travis.yml

```diff
@@ -2,13 +2,13 @@
 # * http://www.objc.io/issue-6/travis-ci.html
 # * https://github.com/supermarin/xcpretty#usage
 
-osx_image: xcode7.3
+osx_image: xcode8
 language: objective-c
 # cache: cocoapods
 # podfile: Example/Podfile
-# before_install:
-# - gem install cocoapods # Since Travis is not always on latest version
+before_install:
+- gem install cocoapods --pre # Since Travis is not always on latest version
```

Comment:
I think that now when Cocoapods 1.1.x is now released we should no more need the `--pre` flag